src/openms/include/OpenMS/CHEMISTRY/IsotopeDistribution.h

```diff
@@ -224,6 +239,14 @@ namespace OpenMS
     /// convolves the distribution @p input with itself and stores the result in @p result
     void convolveSquare_(ContainerType & result, const ContainerType & input) const;
 
+    /** @brief calculates the fragment distribution for a fragment molecule and stores it in @p result.
+
+        @param fragment_isotope_dist the isotopic distribution of the fragment (as if it was a precursor).
+        @param comp_fragment_isotope_dist the isotopic distribution of the complementary fragment (as if it was a precursor).
+        @param precursor_isotopes a list of which precursor isotopes were isolated
```

Comment:
This one technically returns void, would you like me to add a @return nothing, or describe what ends up getting stored in the "result" variable?